parmed/amber/netcdffiles.py

```diff
@@ -262,14 +262,18 @@ def coordinates(self, stuff):
 
     @property
     def velocities(self):
-        vels = self._ncfile.variables['velocities'][:]
-        return (vels.reshape((-1, self.atom, 3)) * self.velocity_scale)
+        if self.hasvels:
```

Comment:
Replace this with:

```python
if 'velocities' in self._ncfile.variables:
```

Alternatively, you can add a `hasvels` property like so:

```python
@property
def hasvels(self):
    return 'velocities' in self._ncfile.variables
```

Reply:
I think the first way makes things easier for now. If `hasvels ` is a property, we will probably need a setter and something like `_hasvels` since `hasvels` is set as an attribute in `open_new` method.

Reply:
hasvels shouldn't be an attribute in `open_new`, probably (is it even *used* anywhere?).  But I agree -- this is the easiest way to go, I think.